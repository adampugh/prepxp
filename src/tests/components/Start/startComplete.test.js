import React from "react";
import { shallow } from "enzyme";

import StartComplete from "../../../components/Start/startComplete";

describe("<StartComplete />", () => {
    it("should match snapshot", () => {
        const wrapper = shallow(<StartComplete />);
        expect(wrapper).toMatchSnapshot();
    });
});