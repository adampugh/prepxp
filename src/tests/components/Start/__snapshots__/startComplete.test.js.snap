// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StartComplete /> should match snapshot 1`] = `
<section
  className="startComplete__div"
>
  <div
    className="container"
  >
    <div
      className="startComplete__grid"
    >
      <div
        className="startComplete__grid__text"
      >
        <h1>
          Questions complete
        </h1>
        <p>
           Well done you have completed this list, good luck in your upcoming interviews!
        </p>
        <div
          className="startComplete__buttons"
        >
          <button
            className="btn"
          >
            Dashboard
          </button>
        </div>
      </div>
      <img
        alt="people conducting an interview"
        src="interview2.png"
      />
    </div>
  </div>
</section>
`;
